components/Cards/index.js

```diff
@@ -22,7 +22,17 @@
 axios
  .get('https://lambda-times-backend.herokuapp.com/articles')
  .then(results => {
-     console.log(results.data)
+     const res = results.data.articles;
+    //  console.log()
+     Object.keys(res).forEach(param => {
```

Comment:
I would use descriptive parameter and variable names when writing your code. On first glance I could not really tell what the function of these lines of code was. If it wasn't for the fact that this was in the cards folder, I wouldn't even know that we were messing with the cards component here.

Reply:
I made the changes for the readme did you want me to add more? I also cleaned it up so that its easier to read.